lib/components/util/accessibility-routing.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Determine if an itinerary has accessibility scores
+ */
+export const itineraryHasAccessibilityScores = (itinerary) => {
```

Comment:
For the i18n work, I was organising util functions into `lib/util`, and utility components like FormattedDuration into `lib/components/util`. Should these be moved to lib/util?

Reply:
Good point, if the function is used from multiple files then it should go to `lib/util`, if not, it should become private to this file.

Reply:
Sorry I was merging too many things this morning and accidentally forgot about this! Very sorry. The fix is in #487